goal_src/jak3/levels/city/common/ff-squad-control.gc

```diff
@@ -672,6 +672,23 @@
                     (-> a1-13 guard-info-normal min-target-count)
                     )
                   )
+            ;; og:preserve-this peaceful city cheat
+            (#when PC_PORT
+              (when (and (pc-cheats? (-> *pc-settings* cheats) city-peace) 
+                         (not (task-node-open? (game-task-node city-port-fight-resolution)))
+                         (not (task-node-open? (game-task-node city-port-assault-resolution)))
+                         (not (task-node-open? (game-task-node city-protect-hq-resolution)))
+                         )
+                (dotimes (guard-info-type 3) ;; normal/war/mission
+                  (when (= (-> a1-13 data guard-info-type object-type) (traffic-type guard-a))
```

Comment:
`wlander-male` can be part of ff-squad-control supposedly, so I'm explicitly checking for `guard-a` here. There is a `guard-b` too but as far as I could tell it's not used?